client/classic/upe/index.js

```diff
@@ -676,9 +677,11 @@ jQuery( function ( $ ) {
 	maybeShowAuthenticationModal();
 
 	// Handle hash change - used when authenticating payment with SCA on checkout page.
-	window.addEventListener( 'hashchange', () => {
+	$( window ).on( 'hashchange', () => {
```

Comment:
This is really just to get rid of a linting issue where ESLint doesn't want us to use the `window` global directly. We can change it back, this line change is definitely not necessary.